src/node_buffer.cc

```diff
@@ -1072,6 +1072,42 @@ void IndexOfNumber(const FunctionCallbackInfo<Value>& args) {
                                 : -1);
 }
 
+void SwapShort(const FunctionCallbackInfo<Value>& args) {
+  Environment* env = Environment::GetCurrent(args);
+  THROW_AND_RETURN_UNLESS_BUFFER(env, args.This());
+  SPREAD_ARG(args.This(), ts_obj);
+
+  if (ts_obj_length % 2 != 0)
+    return env->ThrowRangeError(
+      "Buffer length must be a multiple of 16-bits");
+
+  for (size_t i = 0; i < ts_obj_length; i += 2) {
+    const uint8_t lo = static_cast<uint8_t>(ts_obj_data[i + 0]);
+    const uint8_t hi = static_cast<uint8_t>(ts_obj_data[i + 1]);
```

Comment:
No mathematical operations are being performed on the indexes so I'm not seeing the need for the `static_cast`.

Also, is the `+ 0` to maintain code alignment?